openmc/material.py

```diff
@@ -931,8 +931,11 @@ def from_xml_element(cls, elem):
         mat_id = int(elem.get('id'))
         mat = cls(mat_id)
         mat.name = elem.get('name')
-        if 'temperature' in elem.attrib:
-            mat.temperature = float(elem.get('temperature'))
+
+        temp_node = elem.find("temperature")
```

Comment:
I think I would prefer to handle this the other way -- instead of reading from the subelement, we should just change `to_xml_element` to just write it as an attribute. Temperature seems like a good candidate to be an attribute as opposed to a subelement. The C++ code for reading it will work either way so you won't need to touch that.